pages.zh/common/rsync.md

```diff
@@ -0,0 +1,36 @@
+# rsync
+
+> 一种快速，通用，远程（和本地）文件复制工具
+> 更多信息：<https://man7.org/linux/man-pages/man1/rsync.1.html>
```

Comment:
Missing period?
```suggestion
> 更多信息：<https://man7.org/linux/man-pages/man1/rsync.1.html>。
```

Reply:
Same here.  We put periods after all sentences in the page description, even if they're for the More information link.